### ISSUE TITLE
Is there any way to change hint programmatically?

### ISSUE THREAD

**Issue by mjhassan** [wrapp/floatlabelededittext#46]
I've integrated `floatlabelededittext`, which works fine. But I need to change text(placeholder) and hint on a button click. As I do change the placeholder and hint on button click, hint is not changing. It shows the hint which I placed in XML layout. Is it possible to change hint from button action programmatically? If yes, can anyone please show me how?

**Comment by bsysop** [comment]
+1 for that feature, very important!

**Comment by ShyaamJadhav** [comment]
yes dynamically changed hints are not getting reflected please help!
